content/en/monitors/create/types/ci.md

```diff
@@ -51,15 +52,15 @@ To create a [CI Pipeline monitor][2] in Datadog, use the main navigation: **Moni
 5. Configure the alerting grouping strategy (optional):
    * If the query has a `group by`, multi alerts apply the alert to each source according to your group parameters. An alerting event is generated for each group that meets the set conditions. For example, you could group a query by `@ci.pipeline.name` to receive a separate alert for each CI Pipeline name when the number of errors is high.
 
-{{< img src="monitors/monitor_types/ci_pipelines/define-the-search-query.png" alt="A query for CI Status:Error that is being set to group by Pipeline Name" style="width:80%;" >}}
+{{< img src="monitors/monitor_types/ci/define-the-search-query.png" alt="A query for CI Status:Error that is being set to group by Pipeline Name" style="width:80%;" >}}
```

Comment:
If you do this, the images in JA and FR will be broken for a while. Probably best to leave the directory structure for images as `ci_pipelines`. Copy the images instead of renaming them.